### PR TITLE
[US 313 Task315] Updated backend to read username and password from request body

### PR DESCRIPTION
**Pull request by aarti-kalekar**

### CODE REVIEW

File: .idea/workspace.xml
Comment (shahih25) [inline .idea/workspace.xml:1]: Can you please remove this file?

Comment (aarti-kalekar) [reply .idea/workspace.xml:1]: My bad, yes

Comment (aarti-kalekar) [reply .idea/workspace.xml:1]: Done

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**